app/models.py

```diff
@@ -284,7 +403,6 @@ class Public(UserAccessControl):
     def query_accessible_rows(self, cls, user_or_token, columns=None):
```

Comment:
Do you want to add a deprecation warning or notice in the docs for the query functions?

Reply:
yeah I will add something to the doc strings.